src/migrations/indexes/rulesets_by_official.fauna

```diff
@@ -0,0 +1,13 @@
+// Creates an index for listing either officially supported rulesets or unofficial rulesets
+CreateIndex({
+  name: "rulesets_by_official",
+  source: Collection("rulesets"),
+  terms: [{ field: ["data", "isOfficial"]} ],
+  values: [
+    { field: ["ref"], reverse: true},
```

Comment:
Should fetch by updatedAt to give us the most recent changes

Reply:
Resolved, but the Data Manager needs a way of indexing and sorting by different fields